### PR TITLE
[Stress] MemCheck - add precision field to configs - add precision support

### PR DESCRIPTION
**Pull request by just-sparta**
- *38677*

### CODE REVIEW

File: tests/stress_tests/.automation/memcheck_tests/precommit_configs/desktop_references_config.xml [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
-        <model path="public/vgg16/FP32/vgg16.xml" test="infer_request_inference" device="GPU" vmsize="2892432" vmpeak="3939166" vmrss="2472017" vmhwm="3602924" />
+        <model path="public/mobilenet-ssd/FP32/mobilenet-ssd.xml" precision="FP32" test="create_exenetwork" device="CPU" vmsize="740214" vmpeak="805110" vmrss="129308" vmhwm="129308" />
+        <model path="public/mobilenet-ssd/FP32/mobilenet-ssd.xml" precision="FP32" test="create_exenetwork" device="GPU" vmsize="922147" vmpeak="922147" vmrss="587522" vmhwm="587522" />
+        <model path="public/mobilenet-ssd/FP32/mobilenet-ssd.xml" precision="FP32" test="infer_request_inference" device="CPU" vmsize="1007890" vmpeak="1007890" vmrss="138652" vmhwm="138652" />
+        <model path="public/mobilenet-ssd/FP32/mobilenet-ssd.xml" precision="FP32" test="infer_request_inference" device="GPU" vmsize="1006439" vmpeak="1091636" vmrss="587241" vmhwm="587241" />
+        <model path="public/mtcnn-r/FP32/mtcnn-r.xml" precision="FP32" test="create_exenetwork" device="CPU" vmsize="691589" vmpeak="922864" vmrss="31054" vmhwm="31054" />
```
Comment (vurusovs) [inline tests/stress_tests/.automation/memcheck_tests/precommit_configs/desktop_references_config.xml:8]: path is changed, so it won't be matched for comparison

Comment (just-sparta) [reply tests/stress_tests/.automation/memcheck_tests/precommit_configs/desktop_references_config.xml]: fixed

**Review by vurusovs — APPROVED** [review]

**Comment by asomsiko** [comment]
The commit description seems no related to the change. I can't see int8 or fp16 in the files changed. It is only fp32. @just-sparta could you clarify?

**Review by asomsiko — CHANGES_REQUESTED** [review]
commit message seems unrelated to the change

**Review by asomsiko — APPROVED** [review]

**Pull request merged**